docs/design.md

```diff
@@ -99,7 +99,7 @@ into the target process through a series of steps started by the CLR Profiler DL
 callback, the CLR Profiler DLL takes the following actions:
 
    - If the loaded module is in the set of modules for which there is bytecode instrumentation,
-   or if it's the first non-corelib module or not in one of the special cases,
+   or if it's the first non-core lib module or not in one of the special cases,
```

Comment:
I think corelib is a name used in .NET runtime

Reply:
d6100a3a4df2fb79a4493831d4e83e92f5698de8